tests/scripts/task_python_integration.sh

```diff
@@ -61,12 +61,16 @@ run_pytest cython ${TVM_INTEGRATION_TESTSUITE_NAME}-dso_plugin_module-1 apps/dso
 run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-integration tests/python/integration
 
 # Ignoring Arm(R) Ethos(TM)-U NPU tests in the collective to run to run them in parallel in the next step.
-run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-contrib tests/python/contrib --ignore=tests/python/contrib/test_ethosu --ignore=tests/python/contrib/test_cmsisnn
+run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-contrib tests/python/contrib --ignore=tests/python/contrib/test_ethosu --ignore=tests/python/contrib/test_cmsisnn --ignore=tests/python/contrib/test_opencl
 
 # forked is needed because the global registry gets contaminated
 TVM_TEST_TARGETS="${TVM_RELAY_TEST_TARGETS:-llvm;cuda}" \
     run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-relay tests/python/relay
 
+# OpenCL texture test. Deselected specific tests that fails  in CI
+TVM_TEST_TARGETS="${TVM_RELAY_OPENCL_TEXTURE_TARGETS:-opencl}" \
+    run_pytest ctypes ${TVM_INTEGRATION_TESTSUITE_NAME}-opencl-texture tests/python/relay/opencl_texture --deselect=tests/python/relay/opencl_texture/test_conv2d_nchw_texture.py::test_residual_block
```

Comment:
Instead of deselecting here can you:
1. File a GitHub issue with the test name and some context on why it fails with OpenCL
2. Move the skipping to the test itself via [`pytest.mark.skipIf`](https://docs.pytest.org/en/7.1.x/how-to/skipping.html) and be sure to add a `reason="See <link to apache/tvm GitHub issue>"`? 

That might make it so you don't need to change this script at all

Reply:
But the main thing here is that this test works well locally on adreno devices, so you don’t want to skip it. But in CI environment due to we have nvidia device so different libOpenCL, this test doesn't work properly.

Reply:
Generally we want to exclude some tests only for nvidia devices, but I couldn't find the proper way to exclude tests for nvidia. So maybe there is some way to exclude tests by making sure it doesn't run on nvidia?

Reply:
You can do whatever logic is necessary in the condition for the `skipIf` so that it's only skipped in the specific cases you want (e.g. here maybe just `skipIf(tvm.testing.utils.IS_IN_CI, reason="...")`). Also in `tvm/testing/utils.py` take a look at `requires_cuda` or `uses_gpu` as they may already do what you need

Reply:
Added skipif instead of deselect. Thanks for the suggestion!